app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -0,0 +1,138 @@
+package org.mifos.selfserviceapp.ui.activities;
+
+
+import android.os.Bundle;
+import android.support.design.widget.NavigationView;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.GravityCompat;
+import android.support.v4.widget.DrawerLayout;
+import android.support.v7.app.ActionBarDrawerToggle;
+
+import android.view.MenuItem;
+import android.view.View;
+import android.support.v7.widget.Toolbar;
+
+import org.mifos.selfserviceapp.R;
+import org.mifos.selfserviceapp.ui.fragments.ClientListFragment;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * @author Vishwajeet
+ * @since 14/07/2016
+ */
+
+public class HomeActivity extends BaseActivity implements NavigationView.OnNavigationItemSelectedListener {
+
+    private NavigationView mNavigationView;
+    private DrawerLayout mDrawerLayout;
+
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        getActivityComponent().inject(this);
+
+        setContentView(R.layout.activity_home);
+
+        ButterKnife.bind(this);
+
+        if (toolbar != null) {
+            setSupportActionBar(toolbar);
+            toolbar.setTitle(getString(R.string.home));
+        }
+
+        replaceFragment(new ClientListFragment(), false, R.id.container);
```

Comment:
Don't create fragments like this. 
Use getInstance() method to create Fragments.